01-Arth-Team-Task-automation-using-python.py

```diff
@@ -1,17 +1,27 @@
 import os
 
-def service(ch):

```

Comment:
The service functionality is to give option to the user and take that option and perform the required action